packages/client/src/pages/SignUp/wizards/ParentWizard/index.js

```diff
@@ -61,6 +63,33 @@ const updateRegisteredChild = (index, data) => {
     parentWizardSignUpData.registeredChildren[index] = mergedChildRegistration;
 }
 
+
+const createStudentModel = (studentData) => {
+    return {
+        name: studentData.studentName,
+        email: studentData.studentEmail,
+        gradeLevel: studentData.selectedGradeLevel,
+        subjects: studentData.selectedSubjects,
+    }
+}
+
+/**
+ * Gets all the data from the from and creates a parent 
+ * object based on the parent schema.
+ * @param {object} parentData - Fields the parent filled out.
+ */
+const createParentModel = (parentData) => {
+    return {
+        name: parentData.parentName,
+        phone: parentData.parentPhoneNumber,
+        timezone: parentData.timeZone,
+        email: parentData.parentEmail,
+        students: parentData.registeredChildren
+            .map((studentData) => createStudentModel(studentData)),
+        role: 'PARENT',
+    }
+}
+
```

Comment:
We should move these functions to a shared file. They may be useful when building the Profile page.